docs/esignet-signup-openapi.yaml

```diff
@@ -91,24 +93,26 @@ paths:
                         errorMessage: invalid_transaction
       servers:
         - url: 'https://signup.collab.mosip.net/v1/signup'
+      x-stoplight:
+        id: ar3yjf7kxwpic
   /registration/generate-challenge:
     post:
       tags:
         - UI
       summary: Generate Challenge Endpoint
       description: |-
-        Endpoint to generate challenge for the input identifier of the user. 
-        For eg: if the provided identifier is phone number, OTP is sent to phone. If the identifier is email, OTP is mailed to the given emailID.
+        This endpoint generates a challenge for the provided user identifier. For example, if the identifier is a phone number, an OTP is sent to that phone. If the identifier is an email address, the OTP is sent to the given email.
 
-        TransactionId is created and stored in the cache to maintain the state of the challenge specific to input user identifier.
+        A transaction ID is created and stored in the cache to maintain the state of the challenge specific to the input user identifier. The system should identify whether the provided identifier is a phone number or an email address. The response header should include a Set-Cookie with the transaction ID.
```

Comment:
We can remove this sentence -> The system should identify whether the provided identifier is a phone number or an email address.